content/docs/csidriver/upgradation/drivers/powerflex.md

```diff
@@ -10,9 +10,9 @@ Description: Upgrade PowerFlex CSI driver
 
 You can upgrade the CSI Driver for Dell PowerFlex using Helm or Dell CSI Operator.
 
-## Update Driver from v2.6 to v2.7 using Helm
+## Update Driver from v2.6 to v2.7.1 using Helm
 **Steps**
-1. Run `git clone -b v2.7.0 https://github.com/dell/csi-powerflex.git` to clone the git repository and get the v2.7.0 driver.
+1. Run `git clone -b v2.7.1 https://github.com/dell/csi-powerflex.git` to clone the git repository and get the v2.7.0 driver.
```

Comment:
should this reflect v2.7.1 at the end of the line as well?

Reply:
done